### PR TITLE
3141- fixing the why section to include more descriptive

### PR DESCRIPTION
**Pull request by adipolak**
3141

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ You can easily integrate it with your existing tools, and feel lakeFS in action
...
 With lakeFS data teams can stop worrying about issues that take away much of their time:
-* Roll back changes to data: recover quickly from bugs and mistakes, even on production data
-* Test and validate data before it's marked production-ready exposed to consumers
-* Travel back in time, across data versions and across collections - for reproducing and debugging data issues
+* Roll back changes to data: recover quickly from bugs and mistakes directly on your production data.
+* Test and validate your data before propagating it to data consumers.
+* Travel back in time, across data versions and across data collections - for reproducing and debugging data issues.
 
-Since lakeFS is compatible with the S3 API, all popular applications will work without modification, by simply adding the branch name to the object path:
+Since lakeFS is compatible with the S3, Azure Blog and GCP object storage API, all popular applications will work without modification, by simply adding the branch name to the object path:
```
Comment (johnnyaug) [inline docs/index.md:42]: Unfortunately this is not true - while we do support all cloud platforms as the underlying storage - our API is compatible with S3 but not with the other two.

File: docs/index.md [hunk 2]
```diff
@@ -35,11 +35,11 @@ You can easily integrate it with your existing tools, and feel lakeFS in action
 ## Why use lakeFS? and what it can do
 
 With lakeFS data teams can stop worrying about issues that take away much of their time:
-* Roll back changes to data: recover quickly from bugs and mistakes, even on production data
-* Test and validate data before it's marked production-ready exposed to consumers
-* Travel back in time, across data versions and across collections - for reproducing and debugging data issues
+* Roll back changes to data: recover quickly from bugs and mistakes directly on your production data.
```
Comment (johnnyaug) [inline docs/index.md:38]: I think the "directly" here is a bit scary.

**Review by johnnyaug — CHANGES_REQUESTED** [review]
Thanks!

File: docs/index.md (hunk 1 above)
Comment (adipolak) [reply docs/index.md:42]: thanks, updating!

File: docs/index.md [hunk 3]
```diff
@@ -35,11 +35,11 @@ You can easily integrate it with your existing tools, and feel lakeFS in action
...
+* Test and validate your data before propagating it to data consumers.
```
Comment (arielshaqed) [inline docs/index.md:39]:
```suggestion
* Test and validate your data before propagating it to consumers.
```

File: docs/index.md [hunk 4]
```diff
@@ -35,11 +35,11 @@ You can easily integrate it with your existing tools, and feel lakeFS in action
...
+* Travel back in time, across data versions and across data collections - for reproducing and debugging data issues.
```
Comment (arielshaqed) [inline docs/index.md:40]: Maybe put the value first?

```suggestion
* Reproduce and debug data issues by traveling back in time, across data versions and across data collections.
```

**Review by arielshaqed — APPROVED** [review]
NEAT!

(All changes are merely suggestions!)

**Comment by adipolak** [comment]
Thanks! Updated according to suggestions :)

**Review by johnnyaug — APPROVED** [review]

**Pull request merged**